js/c3-plot.coffee

```diff
@@ -470,6 +482,17 @@ class c3.Plot.Zoomable extends c3.Plot
             @trigger 'restyle', true
         return if domain[0]==extent[0] and domain[1]==extent[1] then null else new_domain
 
+     pan: (dy) =>
```

Comment:
Do you expect to persist the vertical panning state at the app level?

Reply:
Since we can also pan horizontally, then to maintain external state we would also need to pass in `dx`

Reply:
Add a comment that `dx` would be handled by the `extent` in `focus`